README.md

```diff
@@ -25,21 +24,12 @@ Install all required dependencies
 pip install -r requirements.txt
 ```
 
-Create database, superuser and run the server, should be available on http://localhost:8000
+Create database, superuser and run the server, check http://localhost:8000 for swagger
 ```
-cd fundraisee
```

Comment:
I think this shouldnt be removed

Reply:
hmm.. `manage.py` is on `fundraisee-api` folder though

<img width="666" alt="lol" src="https://user-images.githubusercontent.com/17883920/41980054-25bc1312-7a58-11e8-90ca-97c4a60ce199.PNG">

Reply:
oh okok, i initially think its like 
`git clone ...`
`cd fundraisee-api`